packaging/fedora/snapd.spec

```diff
@@ -774,6 +816,11 @@ if [ $1 -eq 1 ] ; then
       systemctl start snapd.socket > /dev/null 2>&1 || :
    fi
 fi
+%if %{with apparmor}
+if [ $1 -eq 1 ] ; then
+    /sbin/apparmor_parser -r /etc/apparmor.d/*snap-confine* > /dev/null 2>&1 || :
+fi
+%endif
```

Comment:
Are we sure this is where we should do this? Also, shouldn't this be before `snapd` is started?

Reply:
This is fine, I was just more concerned about whether the order is sufficient.

Reply:
@Conan-Kudo - I actually moved this down into a `post -n snap-confine` which is more correct. This is probably what you were getting at. I also fixed a bug and confirmed that the packaging DTRT when using `--with apparmor` and not with rpmbuild.